scripts/PrusaColorChange.py

```diff
@@ -0,0 +1,95 @@
+# This PostProcessing Plugin script is released 
+# under the terms of the AGPLv3 or higher
+
+from ..Script import Script
+#from UM.Logger import Logger
+# from cura.Settings.ExtruderManager import ExtruderManager
+
+class PrusaColorChange(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name":"Prusa Color Change",
+            "key": "PrusaColorChange",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "layer_number":
+                {
+                    "label": "Layer",
+                    "description": "At what layer should color change occur. This will be before the layer starts printing. Specify multiple color changes with a comma.",
+                    "unit": "",
+                    "type": "str",
+                    "default_value": 1
+                },
+
+                "initial_retract":
+                {
+                    "label": "Initial Retraction",
+                    "description": "Initial filament retraction distance",
+                    "unit": "mm",
+                    "type": "float",
+                    "default_value": 300.0
+                },
+                "later_retract":
+                {
+                    "label": "Later Retraction Distance",
+                    "description": "Later filament retraction distance for removal",
+                    "unit": "mm",
+                    "type": "float",
+                    "default_value": 3000.0
```

Comment:
Are you sure that a retraction distance of 3000mm is correct? That's 3 metres!

Reply:
That's a bit off. I think I meant 30.0 as the default for extra.